app/javascript/packs/fieldCharacterLimits.js

```diff
@@ -0,0 +1,27 @@
+const user_profile_field_ids = [
```

Comment:
Refactor (blocking): Rename the `fieldCharacterLimits` pack file to something related to the profile settings page, e.g. `profileSettings`. That way future frontend functionality specific to the profile settings page can be housed in there.

For more on webpack and webpacker, see the [Forem frontend documentation on webpacker](https://developers.forem.com/frontend/webpacker).